CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [0.11.0] 2018-06-01
 
 ### Added
+- Changed mechanism used to send closure to child thread in `sched::clone`.
```

Comment:
This description doesn't say why you "changed mechanism".  Did you fix a bug?  Add a feature?  Change just for the heck of it?  Also, it's in the wrong section.  "Added" is for new features.  "Fixed" is for bug fixes.  "Changed" is for backwards-incompatible changes, including bug fixes.